modules/nw-aws-load-balancer-operator.adoc

```diff
@@ -15,7 +15,7 @@ The AWS Load Balancer Operator supports the Kubernetes service resource of type
 
 .Procedure
 
-. You can deploy the AWS Load Balancer Operator on demand from the OperatorHub, by creating a `Subscription` object:
+. You can deploy the AWS Load Balancer Operator on-demand from OperatorHub, by creating a `Subscription` object by running the following command:
```

Comment:
Per ISG, _on-demand_ (with hyphen) is an adjective; _on demand_ (no hyphen) is an adverb. Here, on-demand refers to deploy, and is an adverb. 

```suggestion
. You can deploy the AWS Load Balancer Operator on demand from OperatorHub, by creating a `Subscription` object by running the following command:
```
I'm not sure if the following makes sense, but here is the comparision:
```suggestion
. You can deploy the on-demand AWS Load Balancer Operator from OperatorHub, by creating a `Subscription` object by running the following command:
```